valhalla/baldr/graphconstants.h

```diff
@@ -82,18 +82,22 @@ constexpr uint32_t kMaxDensity = 15;
 // unlimited.
 constexpr uint8_t kUnlimitedSpeedLimit = std::numeric_limits<uint8_t>::max();
 
+// The max assumed speed we know from static data
+constexpr uint32_t kMaxAssumedSpeed = 140; // ~85 MPH
```

Comment:
would it be too messy to change all of these to uint8_t?

Reply:
Sounds good!